.changeset/tame-cows-dress.md

```diff
@@ -0,0 +1,15 @@
+---
+"@kaizen/components": minor
+---
+
+Migrate InlineNotification and GlobalNotication to @kaizen/components
+
+- Migrate `GenericNotification`, `GlobalNotification` and `InlineNotification` from Kaizen legacy repository
+- Update component stories to modern patterns
+- Remove long standing deprecated props:
+ - `autohideDelay` `autohide`
+   - Automatic hiding is an accessibility anti-pattern, violating WCAG '2.2 Enough Time' criteria. This prop will be removed in an upcoming breaking change.
+ - `title` from `InlineNotification``
+   - Use `headingProps` instead
+ - `automationId` and `testId`
+   - These are no longer patterns we use in testing library. Other selectors like `getByRole` or `getByText` should be used instead
```

Comment:
I'll clean this up in the main PR, but because KAIO is technically a brand new package and doesn't have the old components to compare to, we don't really need to detail what changed between the two and we can treat it as a brand new component.

Reply:
Fair enough, just wanted to be thorough in case we were documenting changes between legacy and KAIO versions of components. The main one I was concerned about providing context for was the autohide and autohide delay in case people ask

Reply:
You can document that [here](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3190292705/The+KAIO+Migration+Guide) for now.

Reply:
Ah noice 👍